### DISCUSSION TITLE
A couple of auto crafting table suggestions

### DISCUSSION THREAD

**Discussion by Caridmum** [TechReborn/TechReborn#2520] [Ideas]
I'm working generating uu matter with auto crafting tables as part of the set up, but I'm finding them both slow and annoying to work with. The lack of clear front face makes its mostly trial and error to figure out which sides need to be input and output and the slow speed of crafting means I seem to need more than twice as many crafters as I have cobble generators (vanilla cobble gen with astromine breaker.)

Having a clear front face to the machine would alleviate all the input/output struggle and making the crafters upgradable with overclockers would mean I could use significantly less crafters, and the space they use, for automation.

**Comment by R3DRUMVNE** [comment]
I did a little bit of Photoshop and I hope at least someone will like my idea. I made textures with advanced circuits based on the crafting recipe:

![craft](https://user-images.githubusercontent.com/71280271/94907075-472bc600-04a8-11eb-8d3f-4ed34b4610f0.png)

**Here's what I got:**
* Front side texture:

![auto_crafting_table_front_side](https://user-images.githubusercontent.com/71280271/94908254-12207300-04aa-11eb-87dd-2def8e13a656.png)

* Back side texture:

![auto_crafting_table_back_side](https://user-images.githubusercontent.com/71280271/94908300-23697f80-04aa-11eb-98e1-d4210c0b00db.png)

* Right side texture:

![auto_crafting_table_right_side](https://user-images.githubusercontent.com/71280271/94908340-31b79b80-04aa-11eb-9774-a7c6a6488fa5.png)

* Left side texture:

![auto_crafting_table_left_side](https://user-images.githubusercontent.com/71280271/94908385-409e4e00-04aa-11eb-891c-7c164efd8e8e.png)

* Top side texture:

![auto_crafting_table_top](https://user-images.githubusercontent.com/71280271/94908486-662b5780-04aa-11eb-97f4-9553725ded72.png)

**Comment by R3DRUMVNE** [comment]
I can give you the source of pictures and images 16x16px in png format if you need.
